com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/resolvers/ReferencePattern.java

```diff
@@ -12,7 +12,7 @@
     private String pattern;
     private String patternFormat;
 
-    private ReferencePattern(String pattern, String patternFormat) {
+    ReferencePattern(String pattern, String patternFormat) {

```

Comment:
Why not private?

Reply:
By default, an enum's constructor is private. There is no need to explicitly put "private" here.